contrib/IECoreAlembic/src/IECoreAlembic/AlembicScene.cpp

```diff
@@ -700,7 +956,7 @@ IECore::ConstObjectPtr AlembicScene::readAttribute( const Name &name, double tim
 
 void AlembicScene::writeAttribute( const Name &name, const IECore::Object *attribute, double time )
 {
-	throw IECore::NotImplementedException( "AlembicScene::writeAttribute" );
+	IECore::msg( IECore::Msg::Warning, "AlembicScene::writeAttribute", "Not implemented" );
```

Comment:
Oh, I actually thought we were aiming to merge that before getting a version out. But sure, we can push forward first.